runtime/crab/src/lib.rs

```diff
@@ -309,6 +313,12 @@ frame_support::construct_runtime! {
 		Ethereum: pallet_ethereum = 31,
 		Evm: pallet_evm = 32,
 		BaseFee: pallet_base_fee = 33,
+
+		// Darwinia <> Crab
+		BridgeDarwiniaGrandpa: pallet_bridge_grandpa::<Instance1> = 34,
```

Comment:
Please update the added pallets prefix, 34 is used by the assets pallet in the main branch.